vendors/microchip/boards/secure_element/secure_element.cmake

```diff
@@ -0,0 +1,18 @@
+# -------------------------------------------------------------------------------------------------
+# Amazon FreeRTOS Console metadata
+# -------------------------------------------------------------------------------------------------
+afr_set_board_metadata(ID "Microchip-ATECC608A-with-Windows-Simulator")
+afr_set_board_metadata(DISPLAY_NAME "Microchip ATECC608A with Windows Simulator")
+afr_set_board_metadata(DESCRIPTION " Microchip ATECC608A secure element with Windows Simulator ")
+afr_set_board_metadata(VENDOR_NAME "Microchip")
+afr_set_board_metadata(FAMILY_NAME "ATECC608A")
+afr_set_board_metadata(CODE_SIGNER "AmazonFreeRTOS-Default")
+afr_set_board_metadata(SUPPORTED_IDE "VisualStudio")
+afr_set_board_metadata(RECOMMENDED_IDE "VisualStudio")
+afr_set_board_metadata(IDE_VisualStudio_NAME "Visual Studio")
+afr_set_board_metadata(IDE_VisualStudio_COMPILER "MSVC")
+afr_set_board_metadata(IS_ACTIVE "TRUE")
```

Comment:
might need this additional metadata field as well: https://github.com/aws/amazon-freertos/compare/release-candidate...apregoe:patch-1